README.md

```diff
@@ -11,25 +11,25 @@ This native Node.js module allows you to read managed app preferences on macOS.
 
 ## API
 
-## `prefs.getPreferenceValue(key, type)`
+## `prefs.getPreferenceValue(key, applicationId)`
 
 * `key` String - The preference key to fetch the value of, has to be a valid UTF-8 string.
-* `type` String - The data-type of the preference you are fetching. Can be one of `string`, `integer` or `boolean`.
-* `defaultValue` String | Integer | Boolean (Optional) - If no value is present this value will be returned instead, if no default value is provided this method may return `undefined`.
+* `applicationId` String (Optional) - The identifier of the application whose preferences to search. If no value is provided, the current application's identifier is used.
```

Comment:
Can we not document `applicationId` or even better make it a test-only build thing?

It won't work in sandboxed environments and I don't want people bothering me asking why there app randomly doesn't work / crashes 😆